it/mail/chrome/messenger/phishing-afterload-warning-message.dtd

```diff
@@ -1,28 +1,28 @@
-<!ENTITY safeb.palm.warning.title "Suspected Web Forgery">
+<!ENTITY safeb.palm.warning.title "Sospetta falsificazione web">
 
-<!ENTITY safeb.palm.message.p1 "This page has been reported as a web forgery designed to trick users into sharing personal or financial information. Entering any personal information on this page may result in identity theft or other fraud. &#160;">
-<!ENTITY safeb.palm.message.p1.linkText "Read more &#187;">
-<!ENTITY safeb.palm.p1.linkStatusText "Read more &#133;">
+<!ENTITY safeb.palm.message.p1 "Questa pagina è stata segnalata come falsificazione web realizzata per ingannare gli utenti e spingerli a condividere informazioni personali o finanziarie. Inserire qualsiasi informazione personale su questa pagina potrebbe casusare un furto d'identità o altra frode. &#160;">
```

Comment:
CORRECTIONS NEEDED. 

There is a typo. Please replace "casusare" with "causare".